twitter/tweetarchiver/archiver.py

```diff
@@ -62,16 +62,15 @@ def close(self):
             self.f = None
             try:
                 os.rename(self.fn + '.open', self.fn)
-            except Exception, ex:
+            except Exception as ex:
                 logging.warn('failed to rename %s.open to %s (%s)',
```

Comment:
warn -> warning (you get deprecation warning on `warn`)

Reply:
I have changed it logging.warning. Thank you